rma/models/rma.py

```diff
@@ -0,0 +1,269 @@
+# Copyright 2020 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+RMA_OPERATIONS = [
+    ("refund", "Refund"),
+    ("replace", "Replace"),
+    ("repair", "Repair")
+]
+
+
+class Rma(models.Model):
+    _name = "rma"
+    _description = "RMA"
+    _order = "priority,date desc"
+    _inherit = ["mail.thread"]
```

Comment:
`mail.activity.mixin` and `portal.mixin` should be enabled as well, isn't it?

Reply:
RMAs are still not visible on the portal but I already added this inheritance